packages/trees/frontend/sirius-components-trees/src/views/ExplorerView.tsx

```diff
@@ -11,8 +11,94 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 import { WorkbenchViewComponentProps } from '@eclipse-sirius/sirius-components-core';
+import { makeStyles } from '@material-ui/core/styles';
+import { useEffect, useRef, useState } from 'react';
+import { TreeToolBar } from '../toolbar/TreeToolBar';
+import { FilterBar } from '../trees/FilterBar';
+import { FilterBarState } from './ExplorerView.types';
 import { TreeView } from './TreeView';
 
-export const ExplorerView = (props: WorkbenchViewComponentProps) => (
-  <TreeView {...props} treeId="explorer://" showToolBar={true} enableMultiSelection={true} treeOptions={undefined} />
-);
+const useStyles = makeStyles((theme) => ({
+  treeView: {
+    flexGrow: 1,
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  treeContent: {
+    paddingTop: theme.spacing(1),
+    flexGrow: 1,
+    overflow: 'auto',
+  },
+}));
+
+export const ExplorerView = (props: WorkbenchViewComponentProps) => {
+  const styles = useStyles();
+  const initialState: FilterBarState = {
+    filterBar: false,
+    filterBarText: '',
+    filterBarTreeFiltering: false,
+  };
+  const [state, setState] = useState<FilterBarState>(initialState);
```

Comment:
The state of the `ExplorerView` should not be named `FilterBarState`

Reply:
done